apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -133,7 +133,9 @@ class ScriptEditor extends React.Component {
       projectWidgetVisible: this.props.initialProjectWidgetVisible,
       projectWidgetTypes: this.props.initialProjectWidgetTypes,
       lessonExtrasAvailable: this.props.initialLessonExtrasAvailable,
-      lessonLevelData: this.props.initialLessonLevelData,
+      lessonLevelData:
+        this.props.initialLessonLevelData ||
+        "lesson_group 'lesson group', display_name: 'lesson group display name'\nlesson 'new lesson', display_name: 'lesson display name'\n",
```

Comment:
Had to move setting the lessonLevelData to this initial starting DSL inside ScriptEditor so that when you create a new script you don't end up with it thinking there is a mismatch between the database and the editor.

Reply:
good catch!